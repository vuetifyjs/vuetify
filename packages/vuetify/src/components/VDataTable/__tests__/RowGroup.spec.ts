// @ts-nocheck
/* eslint-disable */

// import RowGroup from '../RowGroup'
import {
  mount,
  Wrapper,
  MountOptions,
} from '@vue/test-utils'

describe.skip('Table RowGroup', () => {
  type Instance = InstanceType<typeof RowGroup>
  let mountFunction: (options?: MountOptions<Instance>) => Wrapper<Instance>
  beforeEach(() => {
    mountFunction = (options?: MountOptions<Instance>) => {
      return mount(RowGroup, options)
    }
  })

  it('should render with "column.summary" slot', () => {
    const wrapper = mountFunction({
      slots: {
        'column.summary': '<div></div>',
      },
    })

    expect(wrapper.html()).toMatchSnapshot()
  })

  it('should render with "row.summary" slot', () => {
    const wrapper = mountFunction({
      slots: {
        'row.summary': '<div></div>',
      },
    })

    expect(wrapper.html()).toMatchSnapshot()
  })
})
