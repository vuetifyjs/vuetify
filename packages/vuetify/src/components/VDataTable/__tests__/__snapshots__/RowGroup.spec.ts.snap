// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table RowGroup should render with "column.summary" slot 1`] = `
<tr class="v-row-group__summary">
  <div>
  </div>
</tr>
`;

exports[`Table RowGroup should render with "row.summary" slot 1`] = `
<div>
</div>
`;
